.github/workflows/shared_deploy.yml

```diff
@@ -0,0 +1,65 @@
+name: Deploy
+
+on:
+  workflow_call:
+    inputs:
+      cloudhub_region:
+        type: string
+        required: true
+      cloudhub_environment:
+        type: string
+        required: true
+      cloudhub_application_name:
+        type: string
+    secrets:
+      CONTD_APP_CLIENT_ID:
+        required: true
+      CONTD_APP_CLIENT_SECRET:
+        required: true
+      CLOUDHUB_BUSINESS_GROUP_ID:
+        required: true
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
```

Comment:
@hoangmirs for the deployment, I think it's ok to let's the previous workflow run there, cancel in the middle might cause some issue